CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Welcome!
+
+We're so glad you're thinking about contributing to the Gov AU UI Kit. The UI Kit is an evolving project and relies on help and feedback to improve.
+
+**Found a bug while using the framework or guide?**
+Let us know by [raising an issue](#submitting-an-issue).
+
+**Have a suggestion for a new feature?**
+If you can reach us on Slack, visit us in #govau-guide and let us know. If you're not on Slack, that's fine, just [open an issue](#submitting-an-issue)
+
+If your suggestion is not already in our roadmap, we'll probably suggest that you make a fork of our repo and then [open a pull request](submitting-a-pr) so that we can review your changes or additions and hopefully merge them in.
+
+We encourage you to read our [Contributor Code of Conduct](https://github.com/AusDTO/gov-au-ui-kit/blob/master/code_of_conduct.md). By ensuring that all staff and contributors follow this guide we can maintain an inclusive and friendly environment.
+
+Finally, if you're unsure about anything, just ask — or submit your issue or pull request anyway. The worst that can happen is we'll politely ask you to change something. We appreciate all friendly contributions.
+
+## Guidelines
+
+### <a name="submitting-an-issue">Submitting an issue</a>
+
+### <a name="submitting-a-pr">Submitting a pull request</a>
+
+To help us get a better understanding of the issue you're submitting, follow our ISSUE TEMPLATE and the guidelines it describes.
+
+### Submitting a pull request
+
+Here are a few guidelines to follow when submitting a pull request:
+
+1. Create a GitHub account or sign in to your existing account.
+1. Fork this repo into your GitHub account (or just clone it if you're an AusDTO team member). Read more about forking a repo here on GitHub:
+[https://help.github.com/articles/fork-a-repo/](https://help.github.com/articles/fork-a-repo/)
+1. Create a branch that lightly defines what you're working on, prefixed with '/feature' (for example, feature/add-styles).
+1. Ensure that your contribution passes a `gulp build.prod`.
+1. Once you're ready to submit a pull request, fill out the PULL REQUEST template provided.
+1. Submit your pull request against the `develop` branch.
+
+Have questions or need help with setup? Open an issue here [https://github.com/AusDTO/gov-au-ui-kit/issues](https://github.com/AusDTO/gov-au-ui-kit/issues).
+
+## Licenses and attribution
+
+### A few parts of this project are not in the public domain
```

Comment:
Our own project is MIT-licensed, as are Bourbon, Neat, and Normalize — does this impact any edits here?